cves/kernel/CVE-2012-2669.yml

```diff
@@ -448,7 +447,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: If anything, the developer probably did not think to check this input, thinking that a limited number of cases would always be returned. However, there were outliers when it came to how the message's origin was returned/checked. 
```

Comment:
Is there evidence to support the check was planned or regularly used in similar scenarios in the project (making this a lapse) or do you believe this is more of a planning error?

Reply:
No. I just assumed this was something they could have thought about during the planning stages. I figured that developers normally think about what type of input will enter a function/program, and try to consider the edge cases for what might make the program break. Here, that edge case was not accounted for. 

However, I don't really know if it is realistic to consider very obscure edge cases a "planning error."

IDK if that makes sense, but that was my thought process.